google/_async_resumable_media/_helpers.py

```diff
@@ -27,12 +27,12 @@
 
 RANGE_HEADER = u"range"
 CONTENT_RANGE_HEADER = u"content-range"
-RETRYABLE = (
-    common.TOO_MANY_REQUESTS,
-    http_client.INTERNAL_SERVER_ERROR,
-    http_client.BAD_GATEWAY,
-    http_client.SERVICE_UNAVAILABLE,
-    http_client.GATEWAY_TIMEOUT,
+RETRYABLE = (  # ConnectionError is also retried, but is not listed here.
```

Comment:
Can this whole set of errors be factored out into a common location for sync/async?

Reply:
Should be possible, yes. I'll try.